CONTRIBUTING.md

```diff
@@ -61,6 +61,7 @@ below:
 * Eleanor Smith (Met Office, UK)
 * Tomasz Trzeciak (Met Office, UK)
 * Mark Worsfold (Met Office, UK)
+* Lucy Liu (Bureau of Meteorology, Australia)
```

Comment:
Just a quick comment, could this be added in alphabetical order? (The list is currently in alphabetical order by surname.)

Reply:
Thanks - only checked the first names for alphabetical order!

Reply:
No worries 👍